src/components/Practice/FluencyP5.jsx

```
@@ -818,226 +802,245 @@ const FluencyP5 = ({
+            position: "relative",
+            overflow: "hidden",
+            justifyContent: isMobile ? "flex-start" : "center",
           }}
-          onClick={() => setOpen(true)}
-        />
-
-        {/* ✅ Common Modal */}
-        {open && (
-          <div
+        >
+          <audio ref={audioRefs} onEnded={handleAudioEnd} hidden />
```

ESLint

[error] Media elements such as <audio> and <video> must have a <track> for captions.

(jsx-a11y/media-has-caption)